packages/app/src/app/pages/Dashboard/Content/Sandboxes/index.tsx

```diff
@@ -6,9 +6,24 @@ import Filters from './Filters';
 
 import { Container, HeaderContainer, HeaderTitle } from '../elements';
 import { DashboardActions } from './Actions';
+import { ITemplate } from './types';
+
+interface Props {
+  sandboxes: any[];
```

Comment:
@CompuIves Shouldn't this be an array of `Sandbox`es? 🤔

Reply:
It's a different type, but I haven't typed it yet.